### PR TITLE
Update ulimit terminology

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

This pull request (PR) fixes https://github.com/magento/devdocs/issues/8941

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.4/comp-mgr/prereq/prereq_compman-ulimit.html

### CODE REVIEW

File: src/_includes/install/ulimit.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
-### Why to set ulimit
+### Why to set open files ulimit
```
Comment (BarnyShergold) [inline src/_includes/install/ulimit.md:1]: The original heading is bad English grammar
```suggestion
### Why open files ulimit should be set
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Good clarification

File: src/_includes/install/ulimit.md (hunk 1 above)
Comment (EvheniiaKolkhiieva) [reply src/_includes/install/ulimit.md]: Hi @BarnyShergold , thank you for reviewing :)

I have implemented your suggestion, could you please approve this PR?
Thank you!

**Review by BarnyShergold — APPROVED** [review]

**Comment by keharper** [comment]
I would like to see following changes:

* Remove the head3s from the include file
* Change the title of `comp-mgr/prereq/prereq_compman-ulimit.md` to "Set the open files limit"
* Update all cross references to this file. I see the TOC file and two files in the v2.3 directory. The 2.4 versions are symlinks
* Change the head2 in `guides/v2.3/install-gde/install/cli/install-cli-backup.md` to match the second bullet item above (don't change the anchor tag)

**Comment by keharper** [comment]
Forgot to tag @EvheniiaKolkhiieva

**Comment by EvheniiaKolkhiieva** [comment]
Hi @keharper, thank you a lot for reviewing!

I have implemented your suggestions, but I can't resolve the conflict ( I haven't write access), could you please help?

Thank you

**Comment by keharper** [comment]
@EvheniiaKolkhiieva Let's start off by reverting your change to src/guides/v2.4/comp-mgr/prereq/prereq_compman-ulimit.md. It's a symlinked file.

**Comment by EvheniiaKolkhiieva** [comment]
Thank you @keharper, I have resolved the conflict and revert the file. Could you please review and approve this PR?

Thank you!

**Review by keharper — APPROVED** [review]

**Comment by keharper** [comment]
@magento import code to magento-devdocs/devdocs

**Comment by magento-engcom-team** [comment]
@keharper the branch with code successfully imported into`magento-devdocs/devdocs` repository. Branch name: `imported-magento-devdocs-8956`.

**Pull request merged**